package.json

```diff
@@ -24,6 +24,8 @@
     "lint": "aegir-lint",
     "coverage": "gulp coverage",
     "test": "gulp test",
+    "test:node": "npm run test:unit:node",
+    "test:browser": "npm run test:unit:browser",
```

Comment:
For making it "compatible" with other projects that only exposes "test:node" and "test:browser" as scripts.